x-pack/test/functional/apps/observability_logs_explorer/alerts.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from './config';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['observabilityLogsExplorer']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Alerts dropdown menu', () => {
+    before(async () => {
+      await PageObjects.observabilityLogsExplorer.navigateTo();
+    });
+
+    it('should create rule successfully', async () => {
+      await testSubjects.clickWhenNotDisabled('logsExplorerToggleAlertsPopoverButton');
+      await testSubjects.click('logsExplorerCreateRuleMenuItem');
+      await testSubjects.setValue('ruleNameInput', 'Test rule');
+      await testSubjects.click('saveRuleButton');
+      await testSubjects.click('confirmModalConfirmButton');
+      await testSubjects.waitForDeleted('saveRuleButton');
+      await testSubjects.waitForDeleted('confirmModalConfirmButton');
+    });
+
+    it('should create SLO successfully', async () => {
```

Comment:
**question**: apart from the current flow, would make sense adding an assertion for the success toast? In the end it is what tells the user if the SLO is successfully created. (there is something for this assertion in our page object IIRC)

Reply:
I started off waiting for the success toast but then realised that it's more reliable to simply wait for the flyout to be closed on success since SLO flyout displays two different toasts so it would be easy to run into race conditions.

Reply:
I see the point, a follow-up question is what happens when the SLO creation fails? Does the flyout stay open?

Reply:
Good point. UX wise the flyout should stay open so that users can correct any error and retry. We can't just close the flyout and let users start from scratch. 

Having said that I have tested both flows and the Create SLO flow seems to have a couple of bugs:
1.) There is no error handling for when the browser is offline. It just hangs. To recreate open dev tools and set to offline in network panel.
2.) When there is a server side error (e.g. 403) the flyout redirects the user to the SLO app instead of leaving the flyout open. To reproduce this change the role of your test user from superuser to viewer after opening the "Create SLO" flyout and before clicking submit.

I have raised these with the team and will add an additional check to ensure a success toast is displayed.

Thanks for double checking.